typescript-arrays/main.ts

```diff
@@ -0,0 +1,33 @@
+const array: string[] = ['red', 'white', 'blue'];
+const colors: string[] = array;
+
+console.log('value of colors [0]', array[0]);
+console.log('value of colors [1]', array[1]);
+console.log('value of colors [2]', array[2]);
+
+const literal: string = `America is ${array[0]}, ${array[1]}, and ${array[2]}.`;
+
+const firstLiteral: string = literal;
```

Comment:
Similar to earlier comment here, you can make the variable `literal` and pass it to the `console.log` later, but you don't need to assign `literal` to another variable.